packages/PreviewCard/PreviewCard.jsx

```diff
@@ -93,25 +104,35 @@ const PreviewCard = ({ header, image, body, footer, href, linkComponent, ...rest
           )}
         </ContentContainer>
       </Box>
-    </BoxContainer>
+    </Box>
   )
 
-  return React.createElement(
+  const clickableCard = React.createElement(
     linkComponent || Anchor,
     {
       ...safeRest(rest),
       href,
     },
     innerCard
   )
+
+  return (
+    <BoxContainer isLarge={hasHeaderOrFooter}>
+      {media && media.type === 'video' && renderVideo(media.content)}
+      {clickableCard}
+    </BoxContainer>
+  )
 }
 
 PreviewCard.propTypes = {
   /**
    * It is strongly suggested to provide this prop.
-   * Image component that will appear at the top of the card, above the content section.  Recommended dimensions is 369x269px.
+   * Media component that will appear at the top of the card, above the content section.  Recommended dimensions is 369x269px.
    */
-  image: PropTypes.node,
+  media: PropTypes.shape({
```

Comment:
@SamMolina This is a BREAKING CHANGE. I don't think we should remove the `image` prop, and instead we should deprecate it. We should continue to support the `image` prop but with a console warning that it is deprecated.

@mike-bunce @affansajid @Andrew-K-Lam

Reply:
good catch, thanks